app/models/user.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  VALID_REGEX_EMAIL = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
```

Comment:
accepts '1@yahoo.com' email

Reply:
Thanks @khadzhinov , I managed to exclude dotted beginning and ending of the local- part of the email address.

Reply:
@khadzhinov Do you suggest to use URI::MailTo::EMAIL_REGEXP? I am reading that it's a built in constant in the Ruby Standard library?